### ISSUE TITLE
IDEA should not be matched with IDEA Super

### ISSUE THREAD

**Issue by u-spec-png** [osmlab/name-suggestion-index#7658]
IDEA and IDEA Super are two different brands
![Screenshot_20230116_174302](https://user-images.githubusercontent.com/54671367/212729464-7f9633a6-52fc-4e53-b3f6-5e3fd9ed0c3d.png)

**Comment by UKChris-osm** [comment]
It seems the NSI has "_IDEA_" listed as a `convenience store`, whereas "_IDEA Super_" is a `supermarket`, is this correct, or are both supermarkets?

If this is the case, then I can see why IDEA Super is being suggested for the node marked as a supermarket as it currently has a matchname for it. Either the matchname needs to be removed, or "_IDEA_" needs to be moved into the `supermarket` category as well.

**Comment by u-spec-png** [comment]
> It seems the NSI has "IDEA" listed as a `convenience store`, whereas "IDEA Super" is a `supermarket`, is this correct, or are both supermarkets?

Yes that is correct. IDEA is always a convenience store and IDEA Super is always supermarket.

**Comment by l29ah** [comment]
Here in .me there are plenty of IDEAs that are supermarkets (that is, they have trolleys for customers).
